### PR TITLE
Remove get_int/get_int64 in favor of getInt<>

### PR DESCRIPTION
**Pull request by maflcko**
No need to have several methods which do the same thing.

### CODE REVIEW

**Review by fanquake — APPROVED** [review]
utACK 93720386e7da7976135da7447ed6fd19e6f3c53b

File: test/object.cpp [hunk 1]
```diff
@@ -92,23 +92,30 @@ BOOST_AUTO_TEST_CASE(univalue_typecheck)
     BOOST_CHECK(v1.isNum());
     BOOST_CHECK_THROW(v1.get_bool(), std::runtime_error);
 
+    {
+        UniValue v_negative;
+        BOOST_CHECK(v_nagative.setNumStr("-1"));
```
Comment (fanquake) [inline test/object.cpp]:
```suggestion
        BOOST_CHECK(v_negative.setNumStr("-1"));
```

**Comment by maflcko** [comment]
Thanks, fixed typo in the test

**Review by fanquake — APPROVED** [review]
ACK 462c503aa416cea9cf39dad30a4ff6147ce6f16c

**Pull request merged**